core/utils/big.go

```diff
@@ -44,6 +44,10 @@ func (b *BigFloat) Value() *big.Float {
 // Big stores large integers and can deserialize a variety of inputs.
 type Big big.Int
 
+func New(b *Big) *Big {
+	return NewBig((*big.Int)(b))
+}
+
```

Comment:
This is handy but I am 100% sure we don't mutate the chain ID anywhere so it should be safe to pass the raw pointer around.